### DISCUSSION TITLE
How to change border color of arrow?

### DISCUSSION THREAD

**Discussion by NetFlareSuperAdmin** [floating-ui/floating-ui#2306] [General]
I'd like to know how I can change the border color of an arrow. For example, I'd like to update the arrow below in gray to red:

<img width="138" alt="Screenshot 2023-04-29 at 4 13 45 PM" src="https://user-images.githubusercontent.com/25718513/235292510-050268e9-44fc-4a99-8821-7fcb2eeb42d3.png">

I've tried to set the background color/border color on both the arrow and ::before elements, but none of them worked.

Thanks!

**Comment by atomiks** [comment]
I'm not sure I understand - what library are you using?

**Comment by huijiewei** [comment]
https://floating-ui.com/docs/FloatingArrow#fill

https://floating-ui.com/docs/FloatingArrow#stroke

https://floating-ui.com/docs/FloatingArrow#tailwind-and-utility-css-styling
